src/test/java/api/support/builders/FeeFineOwnerBuilder.java

```diff
@@ -66,6 +68,16 @@ public FeeFineOwnerBuilder withServicePointOwner(List<JsonObject> servicePointOw
     return new FeeFineOwnerBuilder(id, owner, desc, servicePointOwner, defaultChargeNoticeId,
       defaultActionNoticeId);
   }
+
+  public FeeFineOwnerBuilder withServicePointOwner(UUID ... servicePointIds) {
```

Comment:
Does this describe that this user is an owner of fees or fines for that set of service points?

Reply:
Not really. 
The **FeeFineOwner** it is who should receive charges. It contains a list of service points. 
**FeeFine** it is a type of charge, it commonly associated with an owner (except automatic fee/fines, they don't have an owner).
**Account** it is charge of user - it has properties like: fee/fine type, owner, userId, loanId, itemId, amount, etc.

Reply:
> Not really.
> The **FeeFineOwner** it is who should receive charges.

By receive charges, do you mean the person who is responsible for the organisation receiving the payment of the fee or fine?

> It contains a list of service points.

How would you describe the relationship between the owner and a service point?

> **FeeFine** it is a type of charge, 

I think we should explicitly name this the type. I don't really understand why it isn't called that in the source domain. I find it confusing.

> it commonly associated with an owner (except automatic fee/fines, they don't have an owner).

What is the relationship between an owner and a type? Does this mean that the type used for the fee could differ between owners?

> **Account** it is charge of user - it links everything together and has properties like: fee/fine type, owner, userId, loanId, itemId, amount, etc.

I think we should call this the fee, fine or charge in the context of circulation. It isn't really an account.

This may well be a bigger topic and change than for this pull request.

I can ask these terminology questions to Cate and Holly if you would like me to.

Reply:
Having looked at the `Determine the fee/fine owner` section of CIRC-707, I'm wondering why the owner model in circulation needs to know the set of service points at all.

As far as I can tell, the relationship is only used in order to find the appropriate owner for a fee or fine. Is that a sensible understanding?

Reply:
> By receive charges, do you mean the person who is responsible for the organisation receiving the payment of the fee or fine?

I don't think this is a person, it sounds more like a location to which the fee/fine will be transferred (however it is possible to specify several service points for an owner, not sure how it will be processed in this case).

> How would you describe the relationship between the owner and a service point?

An owner has a name and list of service points, and nothing else.

> What is the relationship between an owner and a type? Does this mean that the type used for the fee could differ between owners?

There are two groups of fee/fine types: **automated** and **manual** (non-automated).
**Manual fee/fine types** - these ones you can create via UI. They belong to an owner and must have only one owner. However an owner can have multiple fee/fine types. So, the fee/fine types have to be configured per owner.
**Automated fee/fine types** - this is preconfigured types that are used for automatic charges like `overdue fee fine` or `lost item (processing) fee`. They don't have an association with owner. And owner is set explicitly to account in this case.

I'd appreciate if you ask Cate and Holly.

Reply:
> I don't think this is a person, it sounds more like a location to which the fee/fine will be transferred 

> An owner has a name and list of service points, and nothing else.

Hmm, I think I might be missing some domain understanding. I don't think I understand what an owner is. Maybe I need to ask Holly that question.

> They don't have an association with owner. And owner is set explicitly to account in this case.

Does that mean that for manual fees / fines the owner is decided based upon the type. 

However for automated fees / fines the owner is decided based upon where the fee was changed?

Reply:
Yes, correct.

Reply:
Does circulation's modelling of an owner need the service points?

Is the relationship only used to determine who the correct owner is for an automated fee / fine?

Reply:
Yes, we have to determine the owner for item's home location.

Reply:
Why do we need to model the owner's service points for that?

We have the primary service point for the borrowed item already I believe, why not find the owner using that? Then there is no need for the owner to have the service points after it has been found?

What am I missing?

Reply:
Oh, this builder is used for tests only, not for production. I use this method to build a sample owner for `cd1` test service point.

Reply:
Doh, of course it is, sorry :-(